packages/mermaid/src/diagrams/packet/renderer.ts

```diff
@@ -0,0 +1,94 @@
+import type { Diagram } from '../../Diagram.js';
+import type { PacketDiagramConfig } from '../../config.type.js';
+import type { DiagramRenderer, DrawDefinition, Group, SVG } from '../../diagram-api/types.js';
+import { selectSvgElement } from '../../rendering-util/selectSvgElement.js';
+import type { PacketDB, PacketWord } from './types.js';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const draw: DrawDefinition = (_text, id, _version, diagram: Diagram) => {
+  const db = diagram.db as PacketDB;
+  const config = db.getConfig();
+  const { rowHeight, paddingY, bitWidth, bitsPerRow } = config;
+  const words = db.getPacket();
+  const title = db.getDiagramTitle();
+  const totalRowHeight = rowHeight + paddingY;
+  const svgHeight = totalRowHeight * (words.length + 1) - (title ? 0 : rowHeight);
+  const svgWidth = bitWidth * bitsPerRow + 2;
+  const svg: SVG = selectSvgElement(id);
+
+  svg.attr('width', '100%');
+  svg.attr('viewbox', `0 0 ${svgWidth} ${svgHeight}`);
+
+  for (const [row, packet] of words.entries()) {
+    drawWord(svg, packet, row, config);
+  }
```

Comment:
Maybe rename `row` to `word` since it has been renamed everywhere to `word`?

```suggestion
  for (const [word, packet] of words.entries()) {
    drawWord(svg, packet, word, config);
  }
```